init_users_mysql.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Database connection details
+DB_HOST="localhost"
+DB_PORT="3307"
+DB_NAME="puresynth"
+DB_USER="pUser"
+DB_PASS="pSynth"
+
+# Function to insert a user
+insert_user() {
+    local username="$1"
```

Comment:
Why do you need username and password?

Reply:
I am replicating the previous hardcoded behavior where for persisting (or checking the existence) a user in the db I specified username, password, and email